tools/ci_build/build.py

```diff
@@ -930,7 +930,7 @@ def main():
     if args.use_tensorrt:
         args.use_cuda = True
 
-    if args.build_wheel:
+    if args.build_wheel or args.gen_doc:
         args.enable_pybind = True
```

Comment:
pybind is to be enabled even if user doesn't state it as op schema/kernel def are available to script via Python bindings